docs/conf.py

```diff
@@ -28,7 +28,8 @@
                                   r"\.(?P<minor>\d+)"
                                   r"\.(?P<patch>\d+)"
                                   r"(?P<tag>.*)?'$",
-                                  fp.read(), re.M).groupdict()
+                                  fp.readline().strip(), re.M).groupdict()
```

Comment:
Sorry, but your change parses the first line of `__init__.py` only.
Doc build will be broken if I even add an empty line at the beginning of the file (or comment or docstring).
Could you apply a loop for iteration over the whole file and returning on the first match?

Reply:
@asvetlov I'd just use `ast` instead or relying on regexps

Reply:
Plus it's always possible to get the version from `pkg_resources`/`setuptools`